modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/mushpeople.dm

```diff
@@ -1,3 +1,11 @@
+/datum/species/mush
+	inherent_traits = list(
```

Comment:
if you need to add a trait or remove it, do it modularly without overriding the var, add or remove to it on New